DisguiseUnityRenderStream/Runtime/PipelineAbstraction/BlitExtended.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using Disguise.RenderStream;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Rendering;
+
+class BlitExtended
+{
+    public enum ColorSpaceConversion
+    {
+        None,
+        LinearToSRGB,
+        SRGBToLinear,
+        Max
```

Comment:
Minor: I must admit that's the kind of trick I'm doing in C++, but what about something like this in C#:
```c#
int k_ColorSpaceConversionConstantsCount = Enum.GetValues(typeof(ColorSpaceConversion)).Cast<int>().Max() + 1;
```
Not only it cannot be "badly modified" in the future (by not respecting the order of constants) but more importantly we do not "leak" a Max enum constant that should be used, we can keep k_ColorSpaceConversionConstantsCount private and so keep internal implementation details inside the class.